### DISCUSSION TITLE
Panning and better navigation tools

### DISCUSSION THREAD

**Discussion by theidleguest** [neo4j-labs/arrows.app#114] [Ideas]
As per the title, it would be nice if the app could allow basic interactions like panning and a better zooming capabilities WITH A THREE BUTTONS MOUSE. It is tedious to navigate big graphs!
Thanks!

**Comment by omegahacker** [comment]
Agreed.  Realistically the only view mode possible right now is a continuous fit-to-screen, which means it doesn't scale for large graphs.  Any attempt to zoom in is transient and it seems to revert to fit view very quickly.  Regular canvas zooming and scrolling should be used...
